src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/ContributionFloorFormItem.tsx

```diff
@@ -7,14 +7,24 @@ import { NftFormFields } from './NftRewardTierModal'
 
 export default function ContributionFloorFormItem({
   form,
+  validateDuplicate,
+  initialValue,
 }: {
   form: FormInstance<NftFormFields>
+  validateDuplicate: (floor: number) => boolean
+  initialValue?: number
 }) {
-  const validatecontributionFloorAmount = () => {
+  const validateContributionFloorAmount = () => {
     const value = form.getFieldValue('contributionFloor')
     if (value === undefined || value <= 0) {
       return Promise.reject(t`Amount required`)
     }
+    const isInitialValue = initialValue?.toString() === value
+    // Check this `contributionFloor` isn't already in another tier
+    // - But allow if tier is being edited and its `contributionFloor` hasn't changed
+    if (!validateDuplicate(value) && !isInitialValue) {
+      return Promise.reject(t`A tier for this amount already exists.`)
```

Comment:
Maybe `A tier at this amount already exists`

Reply:
Yep, feel that. Done